### PR TITLE
Array.prototype.at() method added

### PR DESCRIPTION
**Pull request by sumitdethe27**
### Description
Issue: #2660 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+Title: '.at()'
+Description: 'Return the element at a specified index in an array.'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:3]:
```suggestion
Description: 'Returns the element at a specified index in an array.'
```

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.at()`** method in JavaScript is used to return the element at a specified index in an array. The index can be either positive or negative, and negative indices count from the end of the array.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:15]:
```suggestion
The **`.at()`** method in JavaScript is used to return the element at a specified index in an array. The index can be either positive or negative (negative indices count from the end of the array).
```

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:19]:
```suggestion
```pseudo
```

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+array.at(index);
+```
+
+- `index`:The element at the specified `index` is the one intended to be returned.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:23]:
```suggestion
- `index`: The index value of the element to be returned.
```

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example 1
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:25]:
```suggestion
## Example 
```

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:26]: There should be an introductory sentence here such as:
The following code demonstrates a basic implementation of `.at()`:

File: content/javascript/concepts/arrays/terms/at/at.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+console.log(firstElement);
+```
+
+This example results in the following output:
+
+```shell
+1
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/at/at.md:40]: The Codecademy infrastructure does not currently support this function, so the codebyte will result in an error. Please delete this example

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@sumitdethe27 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by sumitdethe27** [comment]
@caupolicandiaz i have made changes

**Review by caupolicandiaz — APPROVED** [review]
@sumitdethe27 Just made one small edit regarding the example intro. Looks good for a second review.

**Review by SSwiniarski — APPROVED** [review]
@sumitdethe27 Looks good.

**Comment by SSwiniarski** [comment]
@sumitdethe27 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/arrays/at

image

**Pull request merged**